TrkDiag/src/BkgDiag_module.cc

```diff
@@ -300,9 +309,17 @@ namespace mu2e
       _nactive = _nstereo = _nsactive = _nbkg = 0;
       bool pce = _pgen==2; // primary from a CE
       _nindex=0;
+      double sumEdep(0.);
+      double sqrSumDeltaTime(0.);
+      double sqrSumDeltaX(0.);
+      double sqrSumDeltaY(0.);
       for(auto const& ich : cluster.hits()){
         ComboHit const& ch = _chcol->at(ich);
         BkgClusterHit const& bhit = _bkghitcol->at(ich);
+        sumEdep +=  ch.energyDep();
```

Comment:
edep is measured per straw.  I think this sum should be weighted by the # of straw hits (ch.nStrawHits()